src/pages/Detail/components/OptionList/List/ColorList.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import './ColorList.scss';
+
+class ColorList extends Component {
+  constructor() {
+    super();
+    this.state = {
+      select: false,
+    };
+  }
+
+  selectSizeBtn = () => {
+    this.setState({
+      select: !this.state.select,
+    });
+  };
+  render() {
```

Comment:
```suggestion
  };
  
  render() {
```
구분되는 요소들끼리는 한줄 띄워주세요!

Reply:
넵 수정했습니다!